<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="J1835004000-64 User Story" duration-ms="63201" started-at="2020-12-21T23:10:31Z" finished-at="2020-12-21T23:11:34Z">
    <groups>
    </groups>
    <test name="J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields" duration-ms="32349" started-at="2020-12-21T23:10:31Z" finished-at="2020-12-21T23:11:03Z">
      <class name="sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields@143640d5]" name="beforeClass" is-config="true" duration-ms="4838" started-at="2020-12-21T23:10:31Z" finished-at="2020-12-21T23:10:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="testCase()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields@143640d5]" name="testCase" duration-ms="23952" started-at="2020-12-21T23:10:36Z" finished-at="2020-12-21T23:11:00Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields - Execution Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields - Execution Failed
at sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields.testCase(J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.Exception: J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields: Failed
at simpleQuoting.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields(J1835004000_64.java:43)
at sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields.testCase(J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields.java:56)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields@143640d5]" name="afterClass" is-config="true" duration-ms="3519" started-at="2020-12-21T23:11:00Z" finished-at="2020-12-21T23:11:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- sq.J1835004000_64.J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields -->
    </test> <!-- J1835004000_64_TC01_QuickSaleOpportunity_PreFilledFields -->
    <test name="J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation" duration-ms="30811" started-at="2020-12-21T23:11:03Z" finished-at="2020-12-21T23:11:34Z">
      <class name="sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation@4387b79e]" name="beforeClass" is-config="true" duration-ms="4140" started-at="2020-12-21T23:11:03Z" finished-at="2020-12-21T23:11:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="testCase()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation@4387b79e]" name="testCase" duration-ms="23373" started-at="2020-12-21T23:11:08Z" finished-at="2020-12-21T23:11:31Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation - Execution Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation - Execution Failed
at sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation.testCase(J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.Exception: J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation: Failed
at simpleQuoting.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation(J1835004000_64.java:74)
at sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation.testCase(J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation.java:54)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation@4387b79e]" name="afterClass" is-config="true" duration-ms="3292" started-at="2020-12-21T23:11:31Z" finished-at="2020-12-21T23:11:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- sq.J1835004000_64.J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation -->
    </test> <!-- J1835004000_64_TC02_QuickSaleOpportunity_OpportunityCreation -->
  </suite> <!-- J1835004000-64 User Story -->
</testng-results>
